app/src/main/java/io/homeassistant/companion/android/qs/Tile1Service.kt

```diff
@@ -7,8 +7,11 @@ import androidx.annotation.RequiresApi
 @RequiresApi(Build.VERSION_CODES.N)
 class Tile1Service : TileExtensions() {
 
-    override fun getTile(): Tile {
-        return qsTile
+    override fun getTile(): Tile? {
+        return if (qsTile != null)
+            qsTile
+        else
+            null

```

Comment:
Just changing the return type to optional Tile? should fix this crash. I don't think there is any need to do the null check here, it is the same as writing `return qsTile`

Reply:
Honestly the tile should never be `null` to begin with lol I haven't been able to reproduce it locally so wanted to just ensure the value we are getting instead of a surprise in the next release

Reply:
It is a good change to make as this is implementing a method from a Java class and it could be null.